src/components/Categories.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import '../style.css';
+
+const Categories = () => (
+  <div className="py-3 text-center">
+    <button type="button" className="btn text-white check-progress">CHECK PROGRESS</button>
+  </div>
```

Comment:
- [OPTIONAL] I am making this optional because I know you made the change requested by the previous reviewer but the implementation was not total. Kindly change te text of the button from _**CHECK PROGRESS**_ to _**CHECK STATUS**_. 👍